src/FSharpLint.Core/Text.resx

```diff
@@ -327,4 +327,7 @@
   <data name="RulesFailwithBadUsage" xml:space="preserve">
     <value>Bad usage of failwith: {0}.</value>
   </data>
+  <data name="RulesFavourReRaise" xml:space="preserve">
+    <value>Use reraise() instead of raise.</value>
```

Comment:
this should be more specific because this rule doesn't advocate replacing all uses of `raise` with `reraise`, only the ones that use the same captured exception

Reply:
How about `Use reraise() instead of using raise with the same captured exception.`?

Reply:
Sure but add the word "Rather" before the word "use"